fred2/sexp_tree.cpp

```diff
@@ -4904,7 +4912,7 @@ void sexp_tree::update_help(HTREEITEM h)
 			if (query_operator_argument_type(index, c) == OPF_MESSAGE) {
 				for (j=0; j<Num_messages; j++)
 					if (!stricmp(Messages[j].name, tree_nodes[i].text)) {
-						text.Format("Message Text:\r\n%s", Messages[j].message);
+						text.Format("Message Text:\r\n%s\r\n\r\n%s", Messages[j].message, nodeComment.c_str());
```

Comment:
mmmm... these two blank lines will always be appended.  I would find a way to only append them if `nodeComment` exists, perhaps by adding another `%s` just for them.  For messages that are almost long enough to fill the text box, this could cause them to scroll when they would otherwise fit.